docs/docs/dev/explore/graceful-join-leave.md

```diff
@@ -0,0 +1,113 @@
+# Graceful joining and leaving the mesh
```

Comment:
Should this be part of: https://kuma.io/docs/1.6.x/explore/dpp-on-kubernetes/#lifecycle rather than yet an extra page in explore?

Reply:
Yes! Let me rewrite this